scipy/stats/qmc.py

```diff
@@ -0,0 +1,1112 @@
+"""Quasi-Monte Carlo methods.
+
+Define function to generate sample of points in the unit hypercube.
+
+"""
+from abc import ABC, abstractmethod
+import copy
+import math
+
+import numpy as np
+from scipy.optimize import brute
+from scipy._lib._util import check_random_state
+from scipy.optimize import basinhopping
+from scipy.stats import norm
+from scipy.stats._sobol import (
+    initialize_v, _cscramble, _fill_p_cumulative, _draw, _fast_forward,
+    _categorize, initialize_direction_numbers
+)
+
+__all__ = ['discrepancy', 'Halton', 'OrthogonalLatinHypercube',
+           'LatinHypercube', 'OptimalDesign', 'Sobol',
+           'NormalQMC', 'MultivariateNormalQMC']
+
+
+def scale(sample, bounds, reverse=False):
+    """Sample scaling from unit hypercube to bounds range.
+
+    Parameters
+    ----------
+    sample : array_like (n_samples, k_vars)
+        Sample to scale.
+    bounds : tuple or array_like ([min, k_vars], [max, k_vars])
```

Comment:
not entirely clear from this how the broadcasting across dimensions works. Maybe provide a code example in the docstring?

Reply:
I added an example.